packages/pspy/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=pspy
+pkgver=149.9c63e5d
+pkgrel=1
+pkgdesc='Monitor linux processes without root permissions'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-recon' 'blackarch-misc')
+url='https://github.com/DominicBreuker/pspy'
+license=('GPL3')
+depends=('glibc')
+makedepends=('git' 'go-pie')
+source=("git+https://github.com/DominicBreuker/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+
```

Comment:
I don't know about go, just wondering if it is normal that https://github.com/BlackArch/blackarch-pkgbuilds/blob/08bfbf4a186ff2d4e5784ae06d83cc3f611d5b73/PKGBUILD-go#L29-L30 is not there.

Reply:
Reference to this: 
https://github.com/BlackArch/blackarch/blob/master/packages/gau/PKGBUILD

Reply:
Which one is correct?

cc @noptrix

Reply:
@aancw both are ok.